illumio_connector.py

```diff
@@ -472,23 +478,57 @@ def _handle_create_enforcement_boundary(self, param):
     def _handle_get_workloads(self, param):
         action_result = self.add_action_result(ActionResult(dict(param)))
 
-        enforcement_mode = param["enforcement_mode"].lower()
+        enforcement_mode = (
+            param.get("enforcement_mode").lower()
+            if param.get("enforcement_mode")
+            else None
+        )
+        connectivity = (
+            param.get("connectivity").lower() if param.get("connectivity") else None
+        )
+        labels = self.handle_comma_seperated_string(param.get("labels", ""))
+        managed = True
```

Comment:
The `managed` default should probably be `None` - return both managed and unmanaged workloads.

As below, though, might be better to split this out into its own param.

Reply:
Removed "managed=true", because UI does not return managed workloads.